inventory/dynamic/gcp/none

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this needed?

Reply:
Yes, this is the default inventory for dynamic behavior.